src/apps/explorer/components/TokensTableWidget/index.tsx

```diff
@@ -121,6 +142,14 @@ export const TokensTableWidget: React.FC<Props> = () => {
     setPageOffset(0)
   }, [networkId, setPageOffset])
 
+  useEffect(() => {
+    const handleHashChange = (): void => {
+      setSelectedTab(location.hash === '#batches' ? TABS.BATCHES : TABS.TOKENS)
```

Comment:
if you have an enum for this, you can use it also for checking the condition, so no need to add the string `'#batches'`